google-cloud/docs/toc.json

```diff
@@ -712,6 +712,28 @@
         }
       ]
     },
+    {
+      "title": "Tasks",
+      "type": "google/cloud/tasks",
+      "patterns": ["\/tasks", "google\/api", "google\/protobuf"],
+      "nav": [
+        {
+          "title": "V2beta2",
+          "type": "google/cloud/tasks/v2beta2",
+          "patterns": ["\/tasks\/v2beta2", "google\/api", "google\/protobuf"],
+          "nav": [
+            {
+              "title": "CloudTasksClient",
+              "type": "google/cloud/tasks/v2beta2/cloudtasksclient"
+            },
+            {
+              "title": "Data Types",
+              "type": "google/tasks/v2beta2/datatypes"
```

Comment:
As I mentioned in [this comment on the original PR](https://github.com/GoogleCloudPlatform/google-cloud-ruby/pull/2100#discussion_r190072465), I don't think this path will match the output of the `rake jsondoc` task.

Reply:
Why is that? @blowmage seemed to think it would and I have no real understanding of what this is supposed to be. Is there a regression in the generator or jsondoc?

Reply:
The correct value is `google/cloud/tasks/v2beta2/datatypes`. (The jsondoc output file is `google-cloud-tasks/jsondoc/google/cloud/tasks/v2beta2/datatypes.json`, derived from the title in the `toc_config` in the Rakefile: `title: "Google::Cloud::Tasks::V2beta2::DataTypes"`.

Here is the [DataTypes page title in the first PR](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/f7e83f88b355cbab187b347c6d10d86f265323f2/google-cloud-tasks/Rakefile#L78). The value of `type` needs to match.

The generator output for these values has not matched for some time now, perhaps always. It needs to be fixed.

Reply:
@crwilcox @quartzmo AFAIK the generator never did this and it's something I was hoping to address once DPE assumed all the publishing roles. How does this relate to the other documentation changes that you want to make though? Should we fix this `toc.json` issue or can we do more to move that along?

Reply:
This configuration (and problem) will go away if we change the documentation to standard YARD output.